### PR TITLE
rpc JoinGroup and AddNoteToGroup

### PR DESCRIPTION
**Pull request by maxime-dodin**
#### Description

new rpc AddNoteToGroup and JoinGroup
add description to group model
 
related to issue #25

### CODE REVIEW

File: noted/accounts/v1/groups.proto [hunk 1]
```diff
@@ -58,3 +62,19 @@ message ListGroupMembersRequest {
...
+
+message AddNoteToGroupRequest {
+    string id = 1;
+    string note_id = 2;
+}
+
+message AddNoteToGroupResponse {
+    string id = 1;
+    string member_id = 2;
+}
```
Comment (rojas-diego) [inline noted/accounts/v1/groups.proto:80]: I don't think we should return anything here. The reason why it's considered good practice to return data on create/update operations in REST APIs is to return the latest representation of the object within the API. This is useful when there has been side effect changes or when a consumer of the API needs to update the state of the object. Here the API object is a group so either we should return a `Group group = 1;` or we should return nothing. That's just my opinion so LMKWYT.

**Review by rojas-diego — COMMENTED** [review]
Very nice! Just a quick question, why were modifications made to the `*.py` and `*.go` files? The CI is supposed to run on the main branch only, did you run the CI yourself locally?

File: noted/accounts/v1/groups.proto [hunk 2]
```diff
@@ -22,11 +23,14 @@ service GroupsAPI {
     rpc DeleteGroup(DeleteGroupRequest) returns (DeleteGroupResponse) {}
     rpc UpdateGroup(UpdateGroupRequest) returns (UpdateGroupResponse) {}
     rpc ListGroupMembers(ListGroupMembersRequest) returns (ListGroupMembersResponse) {}
+    rpc JoinGroup(JoinGroupRequest) returns (JoinGroupResponse) {}
+    rpc AddNoteToGroup(AddNoteToGroupRequest) returns (AddNoteToGroupResponse) {}
 }
 
 message CreateGroupRequest {
     string name = 1;
     string owner_id = 2;
```
[the remark this answers is not in the document]
Comment (maxime-dodin) [reply noted/accounts/v1/groups.proto]: je suis d'accord,  j'ai ajouter l'id du groupe à la requête JoinGroup, on utilisera le auth aussi pour identifier l'user qui veux rejoindre

File: noted/accounts/v1/groups.proto (hunk 1 above)
Comment (maxime-dodin) [reply noted/accounts/v1/groups.proto]: j'ai suppr la réponse, pour le moment on à pas besoins de return le `Group` donc j'ai laissé blank

**Pull request merged**